### ISSUE TITLE
Keeping NAPs up to date on the website

### ISSUE THREAD

**Issue by brisvag** [napari/napari#4874]
## 📚 Documentation
With the new versioned webiste from #4635, people will land on `stable` automatically. However, parrt of the intent behind NAPs is to use them to start and track the discussion (as stated in several places in [NAP-0](https://napari.org/stable/naps/0-nap-process.html)), which became harder after these changes because NAPs only appear organically on the `dev` website, while they are version-tied in the `stable` branch.

Indeed, if a user goes on the website right now, [they will only find `NAP-0`](https://napari.org/stable/naps/).

I think we should have some sections of the site (NAPs, probably other community stuff, ...) not be tied to the release schedule.

**Comment by melissawm** [comment]
Another option (just for information) is to note somewhere else on the website that this is the stable version and the development version has more up-to-date information. I'm happy to port the NAPs back, though!

**Comment by jni** [comment]
Yeah the home page is "easy" to make special, but it's harder for the NAPs. I would suggest some kind of banner on that page pointing to the dev version?
